app/controllers/avo/associations_controller.rb

```diff
@@ -152,5 +153,15 @@ def authorize_attach_action
     def authorize_detach_action
       authorize_if_defined "detach_#{@field.id}?"
     end
+
+    private
+
+    def set_related_authorization
+      @authorization = if related_resource
+        related_resource.authorization(user: _current_user)
+      else
+        Services::AuthorizationService.new _current_user
+      end
+    end
```

Comment:
The updates in this file aren't related to this PR. They were committed by mistake.